shared/rust/src/domain/jig/module/body/flashcards.rs

```diff
@@ -27,6 +27,10 @@ pub struct Content {
 pub struct PlayerSettings {
     /// display mode
     pub display_mode: DisplayMode,
+
+    /// swap the display to be primary left vs. right
+    #[serde(default)]
```

Comment:
This is a new field so existing modules data will not have it included yet - we need to set a default.